.github/workflows/deploy-earn-protocol-landing-page-staging.yaml

```
@@ -0,0 +1,125 @@
+                       --build-arg MIXPANEL_KEY=${{ secrets.MIXPANEL_KEY }} \
+                       --build-arg NEXT_PUBLIC_MIXPANEL_KEY=${{ secrets.NEXT_PUBLIC_MIXPANEL_KEY }} \
+                       --cache-from type=registry,ref=189194422115.dkr.ecr.${{ env.AWS_REGION }}.amazonaws.com/buildkit-test:cache \
+                       -t $ECR_REGISTRY/$ECR_REPO_NAME:$SHA_TAG \
+                       -t $ECR_REGISTRY/$ECR_REPO_NAME:$LATEST_TAG \
+                       -t $ECR_REGISTRY/$ECR_REPO_NAME:$ENVIRONMENT_TAG \
+                       ./apps/earn-protocol-landing-page
+          docker push $ECR_REGISTRY/$ECR_REPO_NAME --all-tags
+
+      - name: Update ECS service with latest Docker image
+        id: service-update
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:34: Double quote to prevent globbing and word splitting

(shellcheck)